public_html/wp-content/plugins/camptix/addons/payment-stripe.php

```diff
@@ -994,6 +994,7 @@ protected function clean_metadata( $metadata = array() ) {
 
 			// Trim the key to 40 chars.
 			$key = $this->trim_string( $key, 40, '' );
+			$key = str_replace( array( '[', ']' ), array( '(', ')' ), $key );
```

Comment:
```suggestion
			$key = str_replace( array( '[', ']' ), '', $key );
```

As done with the payment descriptor, I think we should just strip these characters instead of replacing them.